Gemfile

```diff
@@ -32,6 +32,7 @@ gem "therubyracer", "~> 0.12.2", platforms: :ruby
 gem "draper", "~> 2.1.0"
 gem "attribute_normalizer", "~> 1.2.0"
 gem "responders", "~> 2.3.0"
+gem "thor", "0.19.1"
```

Comment:
Maybe we should add a comment to this explaining that newer versions of thor produce command line argument errors, so this is a temporary workaround. We will want to remove this constraint once either rails or thor fixes the issue.